app/views/notes/show.html.erb

```diff
@@ -6,6 +6,7 @@
   <% if @node.has_power_tag('replication') %><div class="alert alert-info"><%= raw t('notes.show.replication') %> <a href="/n/<%= @node.power_tag('replication') %>"><%= raw t('notes.show.replication_link') %></a>.</div><% end %>
   <% if @node.has_power_tag('series') %><div class="alert alert-info"><%= raw t('notes.show.series') %> <a href="/tag/series:<%= @node.power_tag('series') %>"><%= @node.power_tag('series') %></a>.</div><% end %>
   <% if @node.has_power_tag('build') %><div class="alert alert-info"><%= raw t('notes.show.build') %> <a href="/n/<%= @node.power_tag('build') %>"><%= raw t('notes.show.build_link') %></a>.</div><% end %>
+  <% if @node.has_power_tag('alert') %><div class="alert alert-info"> <a href="/n/<%= @node.power_tag('alert') %>"><%= @node.power_tag('alert') %></a>.</div><% end %>
```

Comment:
hey @CleverFool77, the addition of alert is good but we need to show the feature-content inside so it would be something like 

```
<% cache('feature_tags-header', skip_digest: true) do %>
    <%= feature('tags-header') %>
 <% end %>
```

Regarding type of alert, let's ask @jywarren. Thanks @CleverFool77 and nice work!!